util/libcrypto.num

```diff
@@ -5481,3 +5481,23 @@ BIO_f_zstd                              ?	3_2_0	EXIST::FUNCTION:COMP
 d2i_PUBKEY_ex_fp                        ?	3_2_0	EXIST::FUNCTION:STDIO
 d2i_PUBKEY_ex_bio                       ?	3_2_0	EXIST::FUNCTION:
 COMP_zlib_oneshot                       ?	3_2_0	EXIST::FUNCTION:COMP
+OSSL_HPKE_keygen                        ?	3_1_0	EXIST::FUNCTION:
```

Comment:
These need to be 3.2 now.. The 3.1 branch is going to be used for a fips 140-3, so it probably doesnt have new features like HPKE included.

Reply:
fixed